0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3472 @@
+{
```

Comment:
<p>Nitpick:</p><blockquote>an upcoming machine learning framework</blockquote><p>Could instead be</p><blockquote>a new machine learning framework</blockquote><p>given that jax is already quite widely used it doesn't feel "upcoming" to me (but I suppose it is still V0). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Updated. Thanks for feedback! </p>